data/json/items/containers.json

```diff
@@ -449,19 +450,12 @@
     "looks_like": "box_small",
     "description": "A sturdy cardboard box, about the size of a banana box.  Great for packing.",
     "weight": "850 g",
-    "volume": "2100 ml",
-    "//": "Volume is much lower than the actual volume of a box this size because presumably if it's in your pack, it isn't empty and full of air; and if it's in your hands, it's irrelevant what the volume is.",
+    "volume": "13600 ml",
+    "longest_side": "30 cm",
+    "//": "Nested containers are a thing now so these need to be actual volume (30x30x15)",
```

Comment:
For these ones I actually used a real box for the model, so we know the dimensions; 10 inches high by 16 inches wide by 20 inches. So that's 25cm x 40cm x 50cm, 50 L, with a longest side of around 60cm or so when you put stuff in diagonally on the bottom.